client/src/AdvancedSearch.elm

```diff
@@ -91,7 +91,7 @@ mainView route routingMsg formMsg data categories =
                         , checked <| selector data
```

Comment:
This MR is ready for review, this is what it looks like currently:

Note: This "add to cart" button is still under dicussion, so I used the current UX for now. New update for this feature, will be done in #57 .

(Uploaded to slack, due to the video is too big for github)
- https://serokell.slack.com/archives/C092019USF4/p1754441292123829?thread_ts=1754441250.771489&cid=C092019USF4